### PR TITLE
CLI: turn switches to commands

### LINKED ISSUE #88: ascii_art | Implement independant operations as sub commands
The following options don't really modify the creation of ascii images. They're logically independent from that logic. Because of this, we always exit (or return) after completing them. We need to change them into sub-commands.
This is a bit of work and might require a read up of the click library docs.

Operations are:

1. --clock
2. --credits
3. --text & --textinput (They're related, so they should be options of the same sub command)

When this is done, the way we'll show credits (for instance) should be to run: `ascii_art credits`. Meaning 'credits' is a subcommand of 'ascii_art'. And we should be able to run help on it: `ascii_art credits --help`.

See: https://click.palletsprojects.com/en/7.x/commands

### PR DESCRIPTION
**Pull request by szabopeter**
Should fix #88

### CODE REVIEW

File: ascii_art/cli.py [hunk 1]
```diff
@@ -15,46 +20,49 @@
...
-              help='show clock as a colorful animation. resize the terminal or press "q" or "x" to exit the clock.')
 @click.option('--all', is_flag=True, help='convert all supported files')
 @click.option('-c', '--color', type=click.Choice(COLOR_OPTIONS, case_sensitive=False), default='black',
               help='Set output color')
-@click.option('--textinput', is_flag=True, help='Convert Simple Text Into Ascii Text Format, Enter Text After Prompt')
-@click.option('--text', default=None, type=str, help='Convert Simple Text Into Ascii Text Format')
-@click.option('--types', is_flag=True, help='list supported image formats and exit.')
 @click.option('-hr', '--highres', is_flag=True, help='Converts using a wide range of Ascii characters.')
-def main(input_files, reverse, save, output, width, credits, clock, all, color, textinput, text, types, highres):
+def process(input_files, reverse, save, output, width, all, color, highres):
```
Comment (ndeoligence) [inline ascii_art/cli.py:27]: Please change this to `convert` so that it doesn't override the imported method and so it's descriptive of its function.

**Review by ndeoligence — CHANGES_REQUESTED** [review]
@szabopeter good job on getting this done
Please implement the suggested change on the breaking function before I can merge it in

File: ascii_art/cli.py (hunk 1 above)
Comment (szabopeter) [reply ascii_art/cli.py]: Oops, sorry about that.

Comment (szabopeter) [reply ascii_art/cli.py]: I've fixed it.

**Review by ndeoligence — APPROVED** [review]

**Pull request merged**